resources/scripts/components/server/auditlogs/AuditLogHandler.ts

```diff
@@ -0,0 +1,95 @@
+import { ServerAuditLog } from '@/api/server/types';
+
+export function AuditLogHandler (log: ServerAuditLog) {
+    switch (log.action) {
+        case 'server:backup.start':
```

Comment:
Should be "started", same with fail, complete, delete, lock, unlock and restore.*

Reply:
Also, backup_name doesn't seem to be part of the metadata

Reply:
You should install my other pr (#3854) first to resolve this